### PR TITLE
[tellstick] Fix auto-detect handling for Rain and Wind devices

### LINKED ISSUE #7171: [tellstick] Binding cannot handle Wind nor Rain detectors
Tellstick binding cannot handle Oregon WMR200 Wind nor Rain detectors, they are either detected as ordinary temperature/humidity devices or detected multiple time. This is not a only a problem for Oregon manufactured sensors, it is a general problem in the code.

The binding does not use the correct measurement types for the different channel types.

## Expected Behavior
Rain and Wind detectors shall be auto detected correctly with correct type and only once.
The binding does not use the the unit types for measurements, e.g. Angle, Pressure, Speed and Temperature.

## Current Behavior
Rain detectors, they are either detected as ordinary temperature/humidity devices or detected multiple time. Wrong measurement types are in use.

## Possible Solution
Change TellstickDicoveryService.java to check for all channel types for all different devices, making a correct classification and only once per defined device.
Update TelldusDevicesHandler.java to use the correct measurement types.
Update sensor.xml to use the correct measurement types.

## Steps to Reproduce (for Bugs)
1. Try to auto-detect new Tellstick Rain and Wind devices from PaperUI
2. Look in inbox and see that Rain and Wind devices are of wrong type and too many compared to the number you try to detect.

## Context
This needs to be fixed so that Tellstick binding correctly handles Rain and Wind devices for the OpenHAB community. Also that the binding uses the correct measurement types will give more flexibility for OpenHAB users in rules etc.

**NOTE:** The current maintainer jarlebh is no longer active in OpenHAB community.

## Your Environment
* Version used: openHAB 2.5.1
* Tellstick DUO conected via USB
* RPI3 Rasbian Buster installed via OpenHabian
* Oregon WMR200 Weather Station

I will  do a PR with the required changes since I own a WMR200 and have tested a patch for some days.

### PR DESCRIPTION
**Pull request by jannegpriv**
This PR implements a correct auto-detect handling for Rain and Wind devices.
It also uses the correct measurement types for the different channel types.

README has been updated.

Fixes #7171 

I've tested this [jar-file](https://drive.google.com/open?id=1ZxtCDecRDktKebWtl0OSPQGjmq5m7YO7) successfully in my own production environment.

### CODE REVIEW

File: bundles/org.openhab.binding.tellstick/src/main/resources/ESH-INF/thing/sensor.xml [hunk 1]
```diff
@@ -87,59 +91,59 @@
...
-		<state pattern="%d" readOnly="true" />
+		<state pattern="%d mm" readOnly="true" />
 	</channel-type>
 
 	<channel-type id="raintotal">
 		<item-type>Number</item-type>
 		<label>Total Rain</label>
 		<description>Total rain</description>
-		<state pattern="%d" readOnly="true">
+		<state pattern="%d mm" readOnly="true">
```
Comment (cpmeister) [inline bundles/org.openhab.binding.tellstick/src/main/resources/ESH-INF/thing/sensor.xml]: If you are going to use units I would suggest using UoM for this as well.

Comment (jannegpriv) [reply bundles/org.openhab.binding.tellstick/src/main/resources/ESH-INF/thing/sensor.xml]: Agreed, I will fix that too.

File: bundles/org.openhab.binding.tellstick/README.md [hunk 2]
```diff
@@ -139,11 +139,11 @@ PowerSensors ([powersensor]) support the following channels:
...
 
-| Channel Type ID | Item Type | Description                  |
-|-----------------|-----------|------------------------------|
-| windgust        | Number    | This current peak wind gust. |
-| winddirection   | Number    | The current wind direction.  |
-| windaverage     | DateTime  | The current wind avarage.    |
+| Channel Type ID | Item Type    | Description                  |
+|-----------------|--------------|------------------------------|
+| windgust        | Number:Speed | The current peak wind gust.  |
+| winddirection   | Number       | The current wind direction.  |
```
Comment (cpmeister) [inline bundles/org.openhab.binding.tellstick/README.md]:
```suggestion
| winddirection   | Number:Angle | The current wind direction.  |
```

Comment (jannegpriv) [reply bundles/org.openhab.binding.tellstick/README.md]: Updated README with correct measurment units.

**Comment by cpmeister** [comment]
@jannegpriv Can you give me a signoff so I can merge this?

**Comment by jannegpriv** [comment]
> @jannegpriv Can you give me a signoff so I can merge this?

I'm kind of new to this, I've signed off my commits but is there somewhere else I should signoff?

**Comment by cpmeister** [comment]
Nvm, I'm still new to this as well. Couldn't see your signoff in your commit messages but just found them. What you did is just fine.

**Pull request merged**